clients/rook/Dockerfile

```
@@ -0,0 +1,39 @@
+RUN apt-get update && apt-get install -y \
+    clang \
+    libclang-dev \
+    pkg-config \
+    && rm -rf /var/lib/apt/lists/*
+
+# Build from repo context so path dependencies continue to resolve.
+COPY . .
+RUN cargo build --manifest-path clients/rook/Cargo.toml --release --locked --bin rook && \
+    strip clients/rook/target/release/rook
+
+FROM busybox:latest AS prep
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)